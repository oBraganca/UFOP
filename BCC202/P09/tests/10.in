10
MariaLaura 425188 4.7
Bianca 722213 3.8
EnzoGabriel 503509 4.1
JoaoVictor 257646 7.3
JoaoGabriel 429306 6.6
Joao 878674 1.1
Emanuel 788112 7.5
AnaSophia 11052 2.0
Arthur 413413 2.0
Vicente 649612 4.9
