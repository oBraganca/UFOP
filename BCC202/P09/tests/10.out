Joao 878674 1.1
AnaSophia 11052 2.0
Arthur 413413 2.0
Bianca 722213 3.8
EnzoGabriel 503509 4.1
MariaLaura 425188 4.7
Vicente 649612 4.9
JoaoGabriel 429306 6.6
JoaoVictor 257646 7.3
Emanuel 788112 7.5
